cypress/integration/patch/patch-route/tasks-filters.ts

```diff
@@ -75,7 +75,8 @@ const singularStatuses = [
   },
 ];
 
-describe("Tasks filters", function() {
+// TODO: refactor to not wait for / inspect gql query. Use defined data set and test against it.
```

Comment:
Could you make a ticket for these so we dont forget to come back to them in the future.

Reply:
one already exists. https://jira.mongodb.org/browse/EVG-7923